libs/components/forms/src/lib/modules/radio/radio-group.component.spec.ts

```diff
@@ -559,6 +559,33 @@ describe('Radio group component (reactive)', function () {
     await fixture.whenStable();
     await expectAsync(fixture.nativeElement).toBeAccessible();
   });
+
+  it('should set aria-owns as a space-separated list of radio ids', () => {
+    fixture.detectChanges();
+
+    const radioGroupEl: HTMLDivElement =
+      fixture.nativeElement.querySelector('.sky-radio-group');
+
+    expect(radioGroupEl.getAttribute('aria-owns')).toMatch(
```

Comment:
Can this be a little more intentional instead of matching a pattern? e.g. inject or mock the `SkyRadioGroupIdService`, then validate that the `aria-owns` attribute value literally matches the IDs from the service.

Reply:
Good idea. Since the component is generating the IDs using an auto-incrementing integer, I had to inject the `SkyIdService` into the component so I could spy on the ID generation process. Seems to work well. 👍